internal/util/vault.go

```diff
@@ -249,7 +254,7 @@ func (vc *vaultConnection) initCertificates(config map[string]interface{}, secre
 func (vc *vaultConnection) connectVault() error {
 	v, err := vault.New(vc.vaultConfig)
 	if err != nil {
-		return fmt.Errorf("failed creating new Vault Secrets: %w", err)
+		return fmt.Errorf("failed connecting to Vault: %w", err)
```

Comment:
@nixpanic isnt it better to also put `vaultAddress`  from vaultConfig in this error ?

Reply:
I don't think that is useful. The returned error is usually very verbose and contains the URL already ([example](https://github.com/ceph/ceph-csi/pull/2322#issuecomment-886613519))

Reply:
oh. sure , if the error carries it, its good enough. Thanks @nixpanic